.eslintrc

```diff
@@ -0,0 +1,164 @@
+{
+  "parserOptions": {
+    "ecmaVersion": 6,
+    "sourceType": "module",
+    "ecmaFeatures": {
+      "jsx": true // Allows support of JSX, but use of React plugin is required to support React semantics
+    }
+  },
+  "parser": "@typescript-eslint/parser",
+  "plugins": [
+    "node",
+    "promise",
+    "react",
+    "@typescript-eslint"
+  ],
+  "settings": {
+    "react": {
+      "version": "detect"
+    }
+  },
+  "env": {
+    "amd": true,
+    "browser": true,
+    "jquery": true,
+    "node": true,
+    "es6": true, // This enables ES6 global variables AND ES6 syntax
+    "worker": true
+  },
+  "rules": {
```

Comment:
Why not use the recommended ones?

Reply:
This is the default file we put on user's disk to be used when a local project `eslintrc` is not present. We don't use the recommended ones for a few reasons:
1. They all display as errors which we don't want in VS. @minestarks and I discussed this and the justification is that lint issues shouldn't look more severe than JS issues (which can be configured to be turned off).
2. Some may have been left out because we don't deem them important enough to report to the user

Of course users can always change their own version but this is the default shipped so this is what we want the templates to be clean against